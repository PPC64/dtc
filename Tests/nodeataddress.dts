/dts-v1/;

/ {
    foo@0 {
    };

    bar {
        ref = <&{/foo@0}>;
    };
};
