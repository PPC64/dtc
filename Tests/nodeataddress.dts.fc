/dts-v1/;

/ {
	bar {
		foo@0 {
		};
	}
	bar {
		ref = <&{/bar/foo}>;
	};
	// CHECK: Failed to find node with label: /bar/foo
	// CHECK: Possible intended match: /bar/foo@0
};
