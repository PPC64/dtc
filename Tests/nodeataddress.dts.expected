/dts-v1/;

/  {

	foo@0 {

		phandle = <0x1>;
	};
	bar {

		ref = <0x1>;
	};
};
